components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/NestedGeckoView.kt

```diff
@@ -42,6 +45,8 @@ open class NestedGeckoView(context: Context) : GeckoView(context), NestedScrolli
     @VisibleForTesting
     internal var childHelper: NestedScrollingChildHelper = NestedScrollingChildHelper(this)
 
+    private val coroutineScope = rootView.toScope()
```

Comment:
One more question: Do we need to use `rootView` here, or could we just use `toScope()` to create a scope in this view? When *this* view gets detached then we want to cancel the scope, I assume. Since the rootView is outside of fragments (where this view here may be in), it may live much longer than anything we do here.

Reply:
Indeed. Thanks!